### ISSUE TITLE
Kodi specifically made for Quest?

### ISSUE THREAD

**Issue by WilsonBradley** [curif/cumulus-tv-m3u8-loader#28]
Sorry if this is a stupid question, is there a Quest native version of Kodi (Yet) ?

Tanx

**Comment by curif** [comment]
No. Months ago I tested the Android apk in the Quest, it works but was difficult to use because of the quests controls.
